contracts/periphery/Periphery.sol

```diff
@@ -348,7 +367,7 @@ contract Periphery is IPeriphery {
 
         // if margin delta is positive, top up position margin
 
-        if (params.marginDelta > 0) {
+        if (params.marginDelta > 0 || msg.value > 0) {
```

Comment:
Again, would it be simpler and more intuitive to have marginDelta include the value of any ETH?